source/common/network/utility.cc

```diff
@@ -342,5 +342,37 @@ bool Utility::portInRangeList(const Address::Instance& address, const std::list<
   return false;
 }
 
+absl::uint128 Utility::Ip6ntohl(const absl::uint128& address) {
+  // TODO(ccaraman): Support Ip6ntohl for big-endian.
+  ASSERT(isLittleEndian());
+  return flipOrder(address);
+}
+
+absl::uint128 Utility::Ip6htonl(const absl::uint128& address) {
+  // TODO(ccaraman): Support Ip6ntohl for big-endian.
+  ASSERT(isLittleEndian());
+  return flipOrder(address);
+}
+
+absl::uint128 Utility::flipOrder(const absl::uint128& input) {
+  absl::uint128 result{0};
+  absl::uint128 data = input;
+  for (int i = 0; i < 16; i++) {
+    result <<= 8;
+    result |= (data & 0x000000000000000000000000000000FF);
+    data >>= 8;
+  }
+  return result;
+}
+
+bool Utility::isLittleEndian() {
+  int num = 1;
```

Comment:
absl already has a define for this which I think we implicitly include. Can we just use that instead of reimplementing the logic?